.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    target-branch: main
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - dependencies
+
+  - package-ecosystem: "docker"
```

Comment:
Note, for updating the CUDA base image, changes are needed in the Dockerfile so that the `FROM` statement is not parameterized. I'll leave it up to you whether you want to make that change in this PR or in a follow-up.

Reply:
Good point I'll make that change in this PR

Reply:
Done, please check now